docs/kibana.asciidoc

```diff
@@ -0,0 +1,239 @@
+[id="{p}-kibana"]
+== Running Kibana on ECK
+
+The link:k8s-quickstart.html#k8s-deploy-kibana[quickstart] is a good starting point to quickly setup a Kibana instance with ECK.
+This section goes into more detail about how to configure Kibana with ECK.
+
+* <<{p}-kibana-eck-managed-es,Use an Elasticsearch cluster managed by ECK>>
+* <<{p}-kibana-advanced-configuration,Advanced configuration>>
+** <<{p}-kibana-pod-configuration,Pod Configuration>>
+** <<{p}-kibana-configuration,Kibana Configuration>>
+** <<{p}-kibana-scaling,Scaling out a Kibana deployment>>
+* <<{p}-kibana-secure-settings,Secure settings>>
+* <<{p}-kibana-http-configuration,HTTP Configuration>>
+** <<{p}-kibana-http-publish,Load balancer settings and TLS SANs>>
+** <<{p}-kibana-http-custom-tls,Provide your own certificate>>
+** <<{p}-kibana-http-disable-tls,Disable TLS>>
+
+[float]
+[id="{p}-kibana-eck-managed-es"]
+=== Use an Elasticsearch cluster managed by ECK
+
+Connecting Kibana to an Elasticsearch instance managed by ECK is pretty straightforward.
+Here is an example of a minimal Kibana deployment descriptor:
+
+[source,yaml,subs="attributes"]
+----
+apiVersion: kibana.k8s.elastic.co/{eck_crd_version}
+kind: Kibana
+metadata:
+  name: quickstart
+spec:
+  version: {version}
+  count: 1
+  elasticsearchRef:
+    name: quickstart
+    namespace: default
+----
+
+NOTE: `namespace` is an optional parameter that you can use to connect to an Elasticsearch in another namespace.
+
+The Kibana configuration file is automatically setup by ECK to establish a secure connection to Elasticsearch.
+
+[float]
+[id="{p}-kibana-advanced-configuration"]
+=== Advanced configuration
+
+If you already have read the documentation about how to run Elasticsearch on ECK then most of the concepts and the ideas
+described here will sound familiar to you.
+This is because the resource definitions in ECK share the same philosophy when it comes to:
+
+* Customize the Pod configuration
+* Customize the product configuration
+* Manage the HTTP settings
```

Comment:
Consider rewording with:

> If you already looked at the Elasticsearch on ECK documentation, then concepts and ideas
>   described here might sound familiar to you.
>   This is because the resource definitions in ECK share the same philosophy when it comes to:
>  
>   * Customize the Pod configuration
>   * Customize the product configuration
>   * Manage the HTTP settings